### PR TITLE
Corrected sort order of lines in userMapData.json [#9207]

### LINKED ISSUE #9207: Sort school names for userMapData.json
As per the issues discussed in #9190 , someone needs to go through userMapData.json and confirm the ordering for all lines with the following rule in mind:

When comparing school names A and B, remove all whitespace and symbols (namely, hyphens) from both names, then sort alphabetically.

If we look at an example from the current file, we have "Northwest High School" followed by "Northwestern College". To determine which should actually be first, remove white space and symbols to be left with "NorthwestHighSchool" and "NorthwesternCollege". Afterwards, compare the two strings. Since the 'h' in "High School" comes after the 'e' in "ern", we need to swap the two. 

Repeat this for all lines in the file.

### PR DESCRIPTION
**Pull request by srishti-learner**
Corrected the order of lines in userMapData.json according to the sorting requirement.
Fixes #9207 

**Outline of Solution**
The order was maintained according to the requirement but spellings etc. has not been corrected. Similar to hyphen, commas were also considered.

### CODE REVIEW

**Comment by kollonqv** [comment]
Hi @srishti-learner , it seemst that we have contributed to the same problem. Could you please check the pull request _[#9207] Confirm sorted order of school names for userMapData.json #9211_. Is the order correct and are there any missing schools? If the order is correct, we could use that one instead so you do not need to add the missing schools that were accidentally removed. However, I only considered hyphens, not commas or other symbols in the solution.

**Comment by cbb692** [comment]
Sorry about the spam. Github had a server issue yesterday so everything got messed up as a result. Let me know if you have any questions about my comment(s) though

File: src/main/webapp/data/userMapData.json [hunk 1]
```diff
@@ -34,27 +34,25 @@
...
       "Morling College",
       "Murdoch University",
       "Narangba Valley SHS",
       "Photography Studies College",
       "Queensland University of Technology",
       "RMIT University",
+      "Star of the Sea Catholic College",
       "St Catherine's Catholic College",
       "St Ives High School",
-      "St Joseph's School Northam",
```
[the remark this answers is not in the document]
Comment (srishti-learner) [reply src/main/webapp/data/userMapData.json]: added.

**Comment by srishti-learner** [comment]
Hey, @kollonqv . I already did the changes. I didn't see your comment. Let @cbb692 confirm how do they want, with or without considering commas. The required one can be merged then. I have copied some old version of the old mistakenly.

**Comment by srishti-learner** [comment]
@cbb692 I don't know how have I had some older version of file. Have made the changes in my fork.

File: src/main/webapp/data/userMapData.json [hunk 2]
```diff
@@ -87,8 +87,8 @@
       "Annuntia",
       "AP hogeschool Antwerpen",
       "Arteveldehogeschool",
-      "Artevelde University College Ghent",
       "Artevelde University",
+      "Artevelde University College Ghent",
       "BA Botaniek",
       "BA Stassart",
```
Comment (cbb692) [inline src/main/webapp/data/userMapData.json]: Needs to go after "Ba'c'ap..."

File: src/main/webapp/data/userMapData.json [hunk 3]
```diff
@@ -1964,19 +1964,19 @@
       "University of Virginia",
       "University of Washington",
       "University of West Florida",
+      "Unniversity of Washington, Seattle",
```
Comment (cbb692) [inline src/main/webapp/data/userMapData.json]: Take out the 2nd 'n' in Univeristy, and bump it up a line

**Comment by cbb692** [comment]
Seems like you still have some style errors that need to be corrected. Make sure you're running the npm and gradlew checks before pushing.

**Comment by cbb692** [comment]
Just checking in before @xpdavid gives this PR the boot to see if you're still working on this, @srishti-learner

**Comment by srishti-learner** [comment]
I just have checked the json file, not the whole project. You can continue to boot, I feel. Also, unable to build it in my local.

**Comment by srishti-learner** [comment]
@cbb692 I was just working on the JSON file. Maybe he can continue with the boot. Also, issue to build locally.

**Comment by xpdavid** [comment]
Close due to inactivity.

**Pull request closed without merging**